src/connection.js

```diff
@@ -618,6 +640,10 @@ export class Connection {
     [number]: ProgramAccountSubscriptionInfo,
   } = {};
   _programAccountChangeSubscriptionCounter: number = 0;
+  _slotSubscriptions: {
+    [number]: number,
```

Comment:
Looks like this should be `[number]: SlotSubscriptionInfo`, or similar (analog of `AccountSubscriptionInfo`, will need defining.)
This should appease flow in ci...

Reply:
(should be all set, thank you!)